lib/std/target.zig

```diff
@@ -1501,6 +1505,10 @@ pub const Target = struct {
             .other,
             => return result,
 
+            // Operating systems in this list have been verified as not having a standard
+            // dynamic linker path.
+            .haiku => return copy(&result, "/system/runtime_loader"),
```

Comment:
FWIW, while this is correct at the present time, in the future when/if Haiku introduces proper multiarch (i.e. 32-bit on 64-bit) support, there will be multiple dynamic linkers for each arch, and invoking the wrong one will probably cause problems.

Reply:
So should this follow the pattern above (in e.g. linux) of `.haiku =>  switch (self.cpu.arch) {`?

Reply:
for the time being i have left a note regarding multiarch support.